tokens/add_new_token_image.md

```diff
@@ -1,8 +1,8 @@
-# Trust Wallet Token Images
+# Trust Wallet Assets Info
+## Overview
+Hello and welcome to Trust Wallet assets info contribution guide. We appriciate your effort to open-source. Token repository [https://github.com/TrustWallet/tokens](https://github.com/TrustWallet/tokens) (repo) is a source of images used by [Trust Wallet](https://trustwallet.com/) (TW) including:
 
-Token repository [https://github.com/TrustWallet/tokens](https://github.com/TrustWallet/tokens) (repo) source of images for:
-
-1. [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) tokens on Ethereum compatible networks supported by Trust Wallet such as:
+1. [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) tokens on Ethereum compatible networks supported by TW such as:
```

Comment:
`TW` initials only used for once here, I prefer to always use Trust Wallet

Reply:
https://github.com/TrustWallet/developer/pull/32/commits/0f14fc799ccd9621c2f6c976163177c1f0bb1d24